charts/datadog/templates/_daemonset-volumes-linux.yaml

```
@@ -17,6 +17,9 @@
 - hostPath:
     path: /sys/fs/cgroup
   name: cgroups
+- name: pod-resources-socket
+  hostPath:
+    path: {{ .Values.datadog.kubelet.podResourcesSocketDir }}
```

This hostPath will also break GKE Autopilot installations. Can we add a new condition to prevent mounting in both Autopilot and GDC installations? 

```suggestion
{{- if and (not .Values.providers.gke.gdc) (not .Values.providers.gke.autopilot) }}
- name: pod-resources-socket
  hostPath:
    path: {{ .Values.datadog.kubelet.podResourcesSocketDir }}
{{- end }}
```
This can go after line 12

---

Added both conditions